k2/csrc/top_sort_test.cu

```diff
@@ -241,15 +244,16 @@ TEST(TopSort, RandomVectorOfFsas1) {
         Fsa host_sorted_i;
         HostTopSort(cpu_fsa_vec_i, &host_sorted_i);
         Fsa sorted_i = cpu_sorted.Index(0, i);
-        K2_LOG(INFO) << cpu_fsa_vec_i;
-        K2_LOG(INFO) << sorted_i;
-        K2_LOG(INFO) << host_sorted_i;
         EXPECT_EQ(properties[i] & gt, gt);
         EXPECT_TRUE(is_top_sorted[i]);
+        if (!IsRandEquivalent(sorted_i, host_sorted_i, false)) {
+          K2_LOG(INFO) << cpu_fsa_vec_i;
+          K2_LOG(INFO) << sorted_i;
+          K2_LOG(INFO) << host_sorted_i;
+        }
         EXPECT_TRUE(IsRandEquivalent(sorted_i, host_sorted_i, false));
```

Comment:
I'd prefer to have a KALDI_LOG(FATAL).... inside the if-statement.    IsRandEquivalent() is slow, so doing it twice is a bit wasteful.

Reply:
mm, as I commented in the last PR about bug fixes in TopSort, these code is just for debugging, I definitely will remove those LOG after debugging. So please ignore this and I will remove those LOG stuff after add tests with `RenumberFsaVec`